src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java

```diff
@@ -1516,22 +1522,22 @@ public boolean configure(StaplerRequest req, JSONObject o) throws FormException
 				}
 			}
 			try {
-				String tokenParam = o.getString("token").trim();
+				String tokenParam = jsonObject.getString("token").trim();
```

Comment:
Sorry, I am not really familiar with net.sf.json, but some libraries return null for getJSONObject() if object doesn't exist. Just want to make sure that we are safe here and String.trim() wouldn't throw Null Pointer exception for some cases

Reply:
I found in the [documentation](http://json-lib.sourceforge.net/apidocs/net/sf/json/JSONObject.html#getString(java.lang.String)) that method `getString(String)` throws the `JSONException` if key is not found. This error is caught thereafter in the code for all the occurrences of usage of that method in `FortifyPlugin` class. Marking this comment as resolved.